projects/rulenode-core-config/src/lib/components/transform/rename-keys-config.component.html

```diff
@@ -1,21 +1,23 @@
-<section [formGroup]="renameKeysConfigForm">
-  <div class="mat-caption" style="margin-bottom: 8px">{{ 'tb.rulenode.rename-keys-in' | translate }}</div>
-  <mat-radio-group formControlName="fromMetadata" fxLayout="row" fxLayoutGap="16px" style="margin-bottom: 16px;">
-    <mat-radio-button [value]="false" color="primary" >
-      <span>{{'tb.rulenode.data' | translate}}</span>
-    </mat-radio-button>
-    <mat-radio-button [value]="true" color="primary">
-      <span>{{'tb.rulenode.metadata' | translate}}</span>
-    </mat-radio-button>
-  </mat-radio-group>
-  <tb-kv-map-config-old
+<section [formGroup]="renameKeysConfigForm" class="tb-form-panel stroked">
+  <div class="tb-form-panel-title" translate>tb.rulenode.rename-keys-in</div>
+  <div fxFlex fxLayoutAlign="center center">
+    <div class="fetch-to-data-toggle">
```

Comment:
Write this code without attributes `fxFlex fxLayoutAlign`

Reply:
Done